commands/run.go

```diff
@@ -101,33 +102,31 @@ var Run = &cobra.Command{
 			checksToRun := make(chan testfile.CheckToRun, len(loadedChecks))
 			checkResults := make(chan checkResult, len(loadedChecks))
 
-			for i := 0; i < runtime.NumCPU(); i++ {
+			n := runtime.NumCPU()
```

Comment:
just curious, does it make a difference?

Reply:
Nope, I was toggling this a bunch to see how the number of workers used was actually impacting things and just never changed it back. It should probably be toggle-able eventually